.circleci/config.yml

```diff
@@ -17,6 +27,14 @@ commands:
           name: UIKit for React test
           command: yarn run test
           no_output_timeout: 15m
+  clone_deployment_script:
+    steps:
+      - run:
+          name: Git - Clone deployment scripts
+          when: always
+          command: |
+            git config --global user.email "sha.sdk_deployment@sendbird.com"
```

Comment:
need to rethink this email, this necessary?

Reply:
You mean this line itself or just sdk_deployment email?

Reply:
sha.sdk_deployment@sendbird.com, > curious why its there

Reply:
I guess the repo that we're trying to clone [here](https://github.com/sendbird/sdk-deployment) is private one so, need to setup a credential by setting the user.email like this.

Reply:
But I think, it's better to hide this info from here. Let me set it as a env variable.

Reply:
Noticed that unlike other chat SDK repositories, our is public. So I save the sdk user email as a secret env variable.
https://github.com/sendbird/sendbird-uikit-react/pull/658/commits/69d705a06c1b957159d56f58138978dcb396d6dc